README.md

```diff
@@ -258,7 +274,6 @@ Example: `nose==1.3.7 moto==0.4.31`
 Platform-specific variants also available:
 `CIBW_TEST_REQUIRES_MACOS` | `CIBW_TEST_REQUIRES_WINDOWS` | `CIBW_TEST_REQUIRES_LINUX`
 
---
```

Comment:
If this is removed, maybe still add a horizontal line or make the "Example YML syntax" as `###` title, since it is a part of the "Options" section, a not of this specific environment variable.

Reply:
done